packages/frontend/store/auth.store.ts

```diff
@@ -20,37 +21,21 @@ type OnboardStatus = {
   isExploreInfoSkipped?: boolean;
 };
 
-type ConnectedState = {
-  status: 'connected';
-  address: string;
+type AuthState = {
+  status: 'initial' | 'connected' | 'disconnected';
+  address: string | undefined;
   ens: string | undefined;
-  balance: Balances;
-  chain: ConnectedChain;
-  provider: ethers.providers.Web3Provider;
-  walletName: string;
+  balance: Balances | undefined;
+  chain: ConnectedChain | undefined;
+  provider: ethers.providers.Web3Provider | undefined;
+  walletName: string | undefined;
```

Comment:
what's the advantage of using `address: string | undefined` vs `address?: string`?

Reply:
None I can think of, quick refactor leftover